packages/lesswrong/lib/collections/languageModelCache/collection.ts

```diff
@@ -0,0 +1,30 @@
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields } from '../../collectionUtils'
+import { ensureIndex } from '../../collectionIndexUtils';
+
+const schema: SchemaType<DbTagFlag> = {
```

Comment:
Should this be `SchemaType<LanguageModelCache>`?

Reply:
Whoops, yes. Actually this whole collection hasn't ended up with any usages (it was aimed at auto-sumarizer which was prototyped and set aside because the result quality wasn't good enough) so I'm just going to take this collection out entirely.

Reply:
Oh, nice